content/docs/02-getting-started/00-alpha.mdx

```diff
@@ -0,0 +1,325 @@
+---
+title: AI SDK 5 Alpha
+description: Get started with the latest Alpha version of the AI SDK.
+---
+
+# AI SDK 5 - Alpha
+
+<Note type="warning">
+  This is an early preview — AI SDK 5 is under active development. APIs may
+  change without notice. Pin to specific versions as breaking changes may occur
+  even in patch releases.
+</Note>
+
+## Alpha Version Guidance
+
+The AI SDK 5 Alpha is intended for:
+
+- Exploration and early prototypes
+- Green-field projects where you can experiment freely
+- Development environments where you can tolerate breaking changes
+
+This Alpha release is **not recommended** for:
+
+- Production applications
+- Projects that require stable APIs
+- Existing applications that would need migration paths
+
+During this Alpha phase, we expect to make significant, potentially breaking changes to the API surface. We're sharing early to gather feedback and improve the SDK before stabilization. Your input is invaluable—please share your experiences through GitHub issues or discussions to help shape the final release.
+
+<Note type="warning">
+  We expect bugs in this Alpha release. To help us improve the SDK, please file
+  bug reports on GitHub. Your reports directly contribute to making the final
+  release more stable and reliable.
+</Note>
+
+## Installation
+
+To install the AI SDK 5 - Alpha, run the following command:
+
+```bash
+# replace with your provider and framework
+npm install ai@alpha @ai-sdk/[your-provider]@alpha @ai-sdk/[your-framework]@alpha
```

Comment:
would that pin the versions?

Reply:
we can update this with a specific version when it is released or add a note yes